pom.xml

```diff
@@ -26,11 +26,10 @@
     <properties>
         <github.url>scm:git:git@github.com:dockstore/swagger-java-zenodo-client.git</github.url>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <dockstore-core.version>1.15.0-beta.0</dockstore-core.version>
+        <dockstore-core.version>1.16.0-alpha.13</dockstore-core.version>
         <maven-failsafe.version>2.21.0</maven-failsafe.version>
-        <maven-surefire.version>2.21.0</maven-surefire.version>
+        <maven-surefire.version>3.0.0-M5</maven-surefire.version>
```

Comment:
Updated this because Junit annotations weren't being honored.

Reply:
I think the "M" denotes testing releases, see "Milestone Releases"

>  A milestone release is appropriate when part of the overall component is fully functioning and the team wants to make it more widely available for testing ... While every effort should be made to ensure the quality of released code, "milestone" releases are essentially provided as-is with no guarantees of stability or maintenance.

https://commons.apache.org/releases/versioning

Try one of the newer final versions https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-surefire-plugin

Reply:
I did not know that the M stood for that. Updated. FYI I got the milestone version by blindly copying what we use in the dockstore/dockstore: https://github.com/dockstore/dockstore/blob/develop/pom.xml#L48, so we should look at updating that one as well.

Reply:
Created a web service PR as well: dockstore/dockstore#5981